x/cash/controller.go

```diff
@@ -13,15 +13,19 @@ type CoinMover interface {
 	MoveCoins(store weave.KVStore, src weave.Address, dest weave.Address, amount coin.Coin) error
 }
 
+// CoinMinter is an interface to create new coins.
+type CoinMinter interface {
```

Comment:
I think it would be better to follow [method plus -er suffix convention for naming](https://golang.org/doc/effective_go.html#interface-names). It's hard to do this with current method name. How about 

```go
type CoinMinter interface { CoinMint(...) error }
type CoinCreator interface { CoinCreate(...) error }
```

Reply:
I'm happy with renaming `IssueCoins` -> `CoinMint` everywhere. Prefer that over `CoinCreate`